lib/BCGen/HBC/BytecodeProviderFromSrc.cpp

```diff
@@ -240,10 +243,15 @@ BCProviderFromSrc::createBCProviderFromSrcImpl(
     runOptimizationPasses(M);
 
   BytecodeGenerationOptions opts{compileFlags.format};
```

Comment:
As described above, we should make the new parameter provide the defaults, and keep the existing code specifying particular fields. When organised this way, the parameter likely doesn't even need to be optional.